libbeat/publisher/client.go

```diff
@@ -76,6 +77,7 @@ func newClient(pub *BeatPublisher) *client {
 		beatMeta: common.MapStr{
 			"name":     pub.name,
 			"hostname": pub.hostname,
+			"version":  version.GetDefaultVersion(),
```

Comment:
Is there any way for a community Beat to have their own version? I think we have a way in other places where the version is involved.